libs/@guardian/source-development-kitchen/src/react-components/ticker/Ticker.tsx

```diff
@@ -0,0 +1,167 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import { textSans15, textSansBold17 } from '@guardian/source/foundations';
+import { useEffect, useState } from 'react';
+import { useIsInView } from './useIsInView';
+import { useTicker } from './useTicker';
+
+interface TickerCopy {
+	countLabel: string;
+}
+
+export interface TickerData {
+	total: number;
+	goal: number;
+}
+
+export interface TickerStylingSettings {
+	headlineColour: string;
+	totalColour: string;
+	goalColour: string;
+	filledProgressColour: string;
+	progressBarBackgroundColour: string;
+}
+
+export interface TickerSettings {
+	currencySymbol: string;
+	copy: TickerCopy;
+	tickerData?: TickerData;
```

Comment:
I know this is how it's modeled elsewhere, but I think we should make this field required now. It means we can remove the default 0 values in the component. It doesn't really make sense to render a ticker with no data

Reply:
Good point!